index.js

```diff
@@ -21,14 +19,25 @@ exports.register = function(server, options, next) {
       })
     }, (err, resp, payload) => {
       if (err) {
-        sever.log(['micro-metrics', 'error'], err);
+        server.log(['micro-metrics', 'error'], err);
         return;
       }
       if (options.verbose) {
         server.log(['micro-metrics', 'track'], { type, tags, value, fields });
       }
     });
   });
+  if (options.logTrack) {
+    server.on('log', (event, tags) => {
+      options.logTrack.forEach((logTrack) => {
```

Comment:
I think it probably makes sense to loop through server tags first since if there aren't any, we just move on.